devel/doctest/Portfile

```diff
@@ -0,0 +1,18 @@
+PortSystem 1.0
```

Comment:
please add the usual modeline, which can be copied from another Portfile or the guide

Reply:
applied `# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4`